client/src/components/ProTx/components/dashboard/DashboardDisplay.js

```diff
@@ -82,42 +77,139 @@ function DashboardDisplay() {
+                        setSelectedGeographicFeature
+                      }
+                    />
+                  </div>
+                  <div className="display-layout-chart">
+                    <MaltreatmentChart
+                      mapType={mapType}
+                      geography={geography}
+                      maltreatmentTypes={maltreatmentTypes}
+                      observedFeature={observedFeature}
+                      year={year}
+                      selectedGeographicFeature={selectedGeographicFeature}
+                      data={data}
+                      showInstructions
+                    />
+                  </div>
+                </div>
+              </>
+            );
+          }}
+        />
+        <Route
+          path={`${protxRoute}/demographics`}
+          render={() => {
+            // observedFeatures (i.e. Demographic Features) only has 2019 data.
+            setYear('2019');
+            setGeography('county');
+            setMapType('observedFeatures');
+            return (
+              <>
+                <DisplaySelectors
+                  mapType={mapType}
+                  geography={geography}
+                  maltreatmentTypes={maltreatmentTypes}
+                  observedFeature={observedFeature}
+                  year={year}
+                  setMaltreatmentTypes={setMaltreatmentTypes}
+                  setObservedFeature={setObservedFeature}
+                  // limitToTopObservedFeatureFields
+                />
+                <div className="display-layout-root">
+                  <div className="display-layout-map">
+                    <MainMap
+                      mapType={mapType}
+                      geography={geography}
+                      maltreatmentTypes={maltreatmentTypes}
+                      observedFeature={observedFeature}
+                      year={year}
+                      data={data}
+                      selectedGeographicFeature={selectedGeographicFeature}
+                      setSelectedGeographicFeature={
+                        setSelectedGeographicFeature
+                      }
+                    />
+                  </div>
+                  <div className="display-layout-chart">
+                    <ObservedFeaturesChart
+                      mapType={mapType}
+                      geography={geography}
+                      maltreatmentTypes={maltreatmentTypes}
+                      observedFeature={observedFeature}
+                      year={year}
+                      selectedGeographicFeature={selectedGeographicFeature}
+                      data={data}
+                      showInstructions
+                    />
+                  </div>
+                </div>
+              </>
+            );
+          }}
+        />
+        <Route
+          path={`${protxRoute}/analytics`}
+          render={() => {
+            setMapType('predictiveFeatures');
```

Comment:
```suggestion
            setMapType('observedFeatures');
```

This is untested but I'm guessing this is the issue where I am seeing wrong values displayed on map.